src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -30,7 +30,42 @@
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <ScrollViewer  
+        <ItemsControl Name="Filters"
+                      Grid.Column="0" Grid.Row="0"
+                      ItemsSource="{Binding Path=Filters}" 
+                      Background="Transparent"
+                      Padding="0,5,0,5">
+            <ItemsControl.ItemsPanel>
+                <ItemsPanelTemplate>
+                    <WrapPanel Orientation="Horizontal" IsItemsHost="True" />
+                </ItemsPanelTemplate>
+            </ItemsControl.ItemsPanel>
+            <ItemsControl.ItemTemplate>
+                <DataTemplate>
+                    <RadioButton Content="{Binding Path=Name, Mode=OneWay}" 
+                                 GroupName="PackageFilters"
+                                 IsChecked="{Binding Path=IsChecked, Mode=TwoWay}"
+                                 Style="{StaticResource {x:Type ToggleButton}}"
+                                 FontSize="10"
+                                 FontFamily="Artifakt Element"
+                                 HorizontalAlignment="Center"
+                                 BorderThickness="1,1,1,1"
+                                 Background="#CCCCCC"
```

Comment:
are these colors we can get from the style dictionary?

Reply:
Yes